lib/core/middleware.js

```diff
@@ -16,7 +16,8 @@ Middleware.auth = function (ctx) {
 
   const { login, callback } = ctx.$auth.options.redirect
   const pageIsInGuestMode = routeOption(ctx.route, 'auth', 'guest')
-  const insidePage = page => normalizePath(ctx.route.path) === normalizePath(page)
+  const insidePage = page =>
```

Comment:
Is this change necessary?

Reply:
Removed, part of a different branch incorrectly included.